app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -134,7 +135,7 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
   private val error_dismiss: WalletButtonView get() = bindingCreditCardPreSelected?.fragmentIabErrorPreSelected?.errorDismiss ?: bindingCreditCardLayout?.fragmentIabError?.errorDismiss!!
 
   // support_error_layout.xml
-  private val error_message: TextView get() = bindingCreditCardPreSelected?.fragmentIabErrorPreSelected?.genericErrorLayout?.errorMessage ?: bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.errorMessage!!
+  private val error_message: TextView get() = bindingCreditCardLayout?.fragmentAdyenError?.errorMessage ?: bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.errorMessage ?: bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.errorMessage!!
```

Comment:
why keep "bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.errorMessage" twice? it does the same thing